hlapipeline/utils/bitmask.py

```diff
@@ -0,0 +1,637 @@
+"""
+A module that provides functions for manipulating bitmasks and data quality
+(DQ) arrays.
+
+:Authors: Mihai Cara (contact: help@stsci.edu)
+
+:License: `<http://www.stsci.edu/resources/software_hardware/pyraf/LICENSE>`_
+
```

Comment:
Question: Is there a reason this module physically exists in the hlapipeline now rather than importing this module from its original package location?